docs/source/faq.rst

```diff
@@ -15,6 +15,15 @@ for larger modules.
 __ http://stackoverflow.com/questions/39295200/can-i-use-functions-imported-from-py-files-in-dask-distributed
 .. _API docs: http://distributed.readthedocs.io/en/latest/api.html#distributed.executor.Executor.upload_file
 
+Can I launch a tasks from a task?
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+This could be while averaging inside another function submitted with
+dask.distributed. For more detail and explanation, see
```

Comment:
Citing one example application seems odd here.  It could be used for many things.  I still think we should just rely on the Table of Contents for this.  Is there a way to improve the main webpage so that people interested in topics like this will the relevant page more easily?